dist/transfer-benefits-schema.json

```diff
@@ -458,8 +458,24 @@
         "spouse",
         "child"
       ]
+    },
+    "vaFileNumber": {
+      "type": "string",
+      "pattern": "^[cC]{0,1}\\d{8}$"
     }
   },
+  "anyOf": [
+    {
+      "required": [
+        "vaFileNumber"
+      ]
+    },
+    {
+      "required": [
+        "relativeSocialSecurityNumber"
```

Comment:
@annekainicUSDS @lihanli 
Just noticed this--shouldn't this be veteranSocialSecurityNumber?

I believe the relative is the applicant (relative of the veteran), but the veteran is the one who can offer a va file # in lieu of a social security number.